### DISCUSSION TITLE
TypeScript compilation error, cannot find corresponding type declarations.

### DISCUSSION THREAD

**Discussion by hsiddiqui1479** [mkosir/express-typescript-typeorm-boilerplate#246] [General]
Hi,
NEED HELP. I tried to add multer and winston but I am getting typescript compilation error. I have added the corresponding types but still getting same error.

![image](https://user-images.githubusercontent.com/17592682/154906668-c622e687-2685-4a44-8761-963ab981c096.png)

**Comment by jassMarok** [comment]
@hsiddiqui1479 you are probably missing type declarations for multer, install them via `npm i @types/multer`

**Reply by hsiddiqui1479** [reply]
Nope, types are installed, tried removing and reinstalling the node_modules. But no luck.

**Comment by jassMarok** [comment]
@hsiddiqui1479 try destroying the current docker container and creating a new one, you would need to do this everytime you install a new dependency

**Reply by jassMarok** [reply]
@mkosir is there a alternative to doing this that you might have come across
